### PR TITLE
[QI2-1133] Docs

### PR DESCRIPTION
**Pull request by Mythir**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -15,14 +15,49 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.8"
+
       - name: Install poetry
         uses: abatilo/actions-poetry@v3
         with:
           poetry-version: "1.3.2"
```
Comment (eliasKA) [inline .github/workflows/release.yaml:22]: Poetry version seems to be outdated, wonder why this is not updated by dependabot (out of scope for this ticket).

File: README.md [hunk 2]
```diff
@@ -1,94 +1,16 @@
...
-```
-
-## License
-
-[Apache License 2.0].
+1. [Installation and login](https://qutech-delft.github.io/pennylane-quantuminspire2/getting_started/installation.html)
+2. [Submitting your first circuit](https://qutech-delft.github.io/pennylane-quantuminspire2/getting_started/submitting.html)
+3. [Example notebooks](https://qutech-delft.github.io/pennylane-quantuminspire2/notebooks/index.html)
 
 [quantum inspire]: https://www.quantum-inspire.com/
```
Comment (eliasKA) [inline README.md:16]: Might be nice to add a link to Pennylane here

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+This project uses Poetry for package management and tox for managing testing/linting/documentation tools.
+
+## Setting up a development environment
+
+Ensure both Poetry and tox installed (using [pipx](https://github.com/pypa/pipx) is recommended):
```
Comment (eliasKA) [inline docs/contributing.md]:
```suggestion
Ensure both Poetry and tox are installed (using [pipx](https://github.com/pypa/pipx) is recommended):
```

**Review by eliasKA — CHANGES_REQUESTED** [review]
Looks good 🚀 

Two small comments

File: docs/contributing.md (hunk 3 above)
Comment (Mythir) [reply docs/contributing.md]: Nice catch! Fixed.

File: README.md (hunk 2 above)
Comment (Mythir) [reply README.md:16]: Good suggestion, done

**Review by eliasKA — APPROVED** [review]

**Pull request merged**